airflow/auth/managers/base_auth_manager.py

```diff
@@ -36,8 +38,9 @@ class BaseAuthManager(LoggingMixin):
     Auth managers are responsible for any user management related operation such as login, logout, authz, ...
     """
 
-    def __init__(self):
+    def __init__(self, app: Flask):
```

Comment:
```suggestion
    def __init__(self, app: Flask) -> None:
```

Reply:
Yeah I can do that but `__init__` methods are a bit special and also everyone know what they are, adding type annotation there (especially to say `None`) seems pointless to me ?

Reply:
Mypy would require an annotation in strict mode, i.o.w. it doesn’t think this is that special, compared to e.g. `self`. But what is special is of course subjective.